src/test/java/net/datafaker/annotations/EmbeddedFakeProviderTest.java

```diff
@@ -0,0 +1,36 @@
+package net.datafaker.annotations;
+
+import net.datafaker.Faker;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EmbeddedFakeProviderTest {
+
+    @Test
+    void shouldGenerateEntityWithInheritance() {
+        var person = Faker.generate(ParentPerson.class);
+
+        assertThat(person).isNotNull();
+        assertThat(person.firstName).isNotNull();
```

Comment:
Instead of checking for `notNull` i guess we could check against real values.
Since seed is same shouldn't it be constant value each time?